Kitodo-DataManagement/src/main/java/org/kitodo/util/Generator.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.util;
+
+import java.util.List;
+
+import org.kitodo.data.database.beans.Folder;
+import org.kitodo.data.database.beans.Task;
+
+/**
+ * An encapsulation to access the generator properties of the folder.
+ */
+public class Generator {
```

Comment:
Please use a more meaningful name here! What is generated?

Reply:
Done. There is nothing generated, this class wraps the generation settings of the CurrentStepForm, so it controlls whether the generator is „on“ or „off“. It was renamed to „GeneratorSwitch“ which should explain it better. Javadoc was extended to explain what the class does.